@here/harp-materials/lib/SolidLineMaterial.ts

```diff
@@ -170,7 +168,7 @@ void main() {
     #endif
 
     #ifdef USE_OUTLINE
-    float outlineDist = pointDist - lineWidth;
+    float outlineDist = pointDist - 1.0;
```

Comment:
is this a typo or intended change of lineWidth for 1.0?

Reply:
As stated in the comment, now everything is relative to ``[lineLength, lineWidth]``, so a previous distance of ``lineWidth`` is now equivalent to ``1.0``.

Reply:
It is exactly the first point on the message and I missed it.... sorry. Please ignore :)

Reply:
Nobody will read this comment in the future. Please add this documentation to the code.